src/main/java/com/example/moviemate/auth/dto/SignUpDto.java

```diff
@@ -0,0 +1,52 @@
+package com.example.moviemate.auth.dto;
+
+
+
+import com.example.moviemate.user.entity.User;
+import com.example.moviemate.user.entity.type.UserType;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import lombok.*;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SignUpDto {
+
+  @Email
+  private String email;
```

Comment:
```suggestion
  @Email
  @Pattern(regexp = "^[A-Za-z0-9_\\.\\-]+@[A-Za-z0-9\\-]+\\.[A-Za-z0-9\\-]+$")
  private String email; // example@example.com
```

이메일같이 회원식별에 사용될 수 있는 값은 이런식으로 정규식 활용해서 정확한 입력값을 유도해도 좋을 것 같아요!

Reply:
좋은 방법인 것 같습니다. 제 프로젝트에도 반영해야겠네요!